### PR TITLE
Storing gobblin-runtime test logs to gobblin-runtime-tests.log

### PR DESCRIPTION
**Pull request by chavdar**
Storing gobblin-runtime test logs to gobblin-runtime-tests.log to helpp debug intermittent failures

### CODE REVIEW

**Comment by chavdar** [comment]
@sahilTakiar @ydai1124 can you review?

File: gobblin-runtime/src/test/resources/gobblin-runtime-log4j.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+log4j.rootLogger=INFO, console, LogFile
+
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss z} %-5p [%t] %c{2} : %m%n
+
+log4j.appender.LogFile=org.apache.log4j.RollingFileAppender
+log4j.appender.LogFile.File=gobblin-runtime-tests.log
```
Comment (ydai1124) [inline gobblin-runtime/src/test/resources/gobblin-runtime-log4j.properties]: Does this file need to be added to `.gitignore`?

**Comment by ydai1124** [comment]
Minor comment. LGTM.

File: gobblin-runtime/src/test/resources/gobblin-runtime-log4j.properties (hunk 1 above)
Comment (chavdar) [inline gobblin-runtime/src/test/resources/gobblin-runtime-log4j.properties]: Good point.

**Comment by ydai1124** [comment]
LGTM

**Comment by abti** [comment]
@chavdar Can you please resolve conflict and update the PR to merge it in?

**Pull request closed without merging**